web/Earthfile

```diff
@@ -36,7 +36,7 @@ build:
   FROM +prepare
   RUN pnpm run create-generated-files
   RUN pnpm version minor
-  RUN --secret SENTRY_AUTH_TOKEN --secret VITE_PUBLIC_ELECTRICITYMAP_PUBLIC_TOKEN=ELECTRICITYMAP_PUBLIC_TOKEN pnpm run build
+  RUN --secret SENTRY_AUTH_TOKEN --secret VITE_PUBLIC_ELECTRICITYMAP_PUBLIC_TOKEN_V9=ELECTRICITYMAPS_APP_PUBLIC_TOKEN_V9 pnpm run build
```

Comment:
I think we need a backend PR to add this to `.drone-earthly.yml`

Reply:
It's in the mono-repo. It's how earthly will get the secrets when it's building the bundle to deploy to prod

Reply:
If anyone is following this and for our future self the token did need to be added to the drone config as well. This has now been done.